ql/src/java/org/apache/hadoop/hive/ql/exec/tez/monitoring/TezJobMonitor.java

```diff
@@ -458,6 +460,18 @@ private void printSummary(boolean success, Map<String, Progress> progressMap) {
     }
   }
 
+  public void endSummary() {
+    console.endSummary();
```

Comment:
is it ok to call `console.endSummary()` if it was never started?

Reply:
yeah, it's safe, just marks that collection can be stopped IF it has started